frontend/tests/unit/components/form/TextInput.spec.ts

```diff
@@ -1,32 +1,79 @@
-import { mount, flushPromises } from '@vue/test-utils';
+import { shallowMount } from '@vue/test-utils';
 
+import PrimeVue from 'primevue/config';
 import TextInput from '@/components/form/TextInput.vue';
 
 describe('TextInput.vue', () => {
-  // TODO: Resolve Cannot read properties of undefined (reading 'config')
-  it.skip('renders', async () => {
-    const wrapper = mount(TextInput, {
+  it('renders', async () => {
+    const wrapper = shallowMount(TextInput, {
       props: {
         name: 'test',
       },
+      global: {
+        plugins: [PrimeVue]
+      }
     });
 
-    const input = wrapper.find('input[name="test"]');
-    expect(input).toBeTruthy();
+    expect(wrapper).toBeTruthy();
   });
 
-  // TODO: Find a way to get an updated DOM after the event change
-  // The validation element is not being rendered
-  it.skip('validates', async () => {
-    const wrapper = mount(TextInput, {
+  it('displays the label', async () => {
+    const wrapper = shallowMount(TextInput, {
       props: {
-        name: 'test'
+        label: 'testlabel',
+        name: 'test',
+      },
+      global: {
+        plugins: [PrimeVue]
+      }
+    });
+
+    const label = wrapper.find('label');
+    expect(label.text()).toBe('testlabel');
+  });
+
+  it('displays the help text', async () => {
+    const wrapper = shallowMount(TextInput, {
+      props: {
+        helpText: 'some help text',
+        name: 'test',
+      },
+      global: {
+        plugins: [PrimeVue]
+      }
+    });
+
+    const help = wrapper.find('small');
+    expect(help.text()).toBe('some help text');
+  });
+
+  it('sets the placeholder text', async () => {
+    const wrapper = shallowMount(TextInput, {
+      props: {
+        name: 'test',
+        placeholder: 'some placeholder text'
+      },
+      global: {
+        plugins: [PrimeVue]
+      }
+    });
+
+    const input = wrapper.getComponent({ name: 'InputText' });
+    expect(input.attributes('placeholder')).toBe('some placeholder text');
```

Comment:
As a mental note, it may be worth letting the toBe value be assigned as a constant like placeholderValue, before the const wrapper declaration. That way, when we do the shallowMount and do a validation, we can reference back on the same constant and know that it's the same value and memory location, instead of copy-pasting the same string in two places.